pack/rpm.mk

```diff
@@ -18,7 +18,13 @@ else
 RPMNAME := $(shell rpmspec -P $(RPMSPECIN) | sed -n -e 's/Name:\([\ \t]*\)\(.*\)/\2/p')
 endif
 
+ifneq ($(shell rpm -E "%{suse_version}"), %{suse_version})
+IS_SUSE := true
+WITH_BACKTRACE := --with backtrace
```

Comment:
It is tarantool-specific option, should not be in packpack.

Reply:
Right, corrected.